### DISCUSSION TITLE
Temperature Growth Rate cannot by logged at the drying phase

### DISCUSSION THREAD

**Discussion by vincentdxu** [artisan-roaster-scope/artisan#830] [Q&A]
Hi, I'm using ver 4.2.6 on Windows 10. I dont know why the temperature growth rase cannot be logged at the drying phase. Does anyone face similar issue? Any solution? Thanks/Vincent

![IMG_4728](https://user-images.githubusercontent.com/9997647/156088653-75099aba-ce10-4775-b019-c740104a75d2.JPG)

**Comment by MAKOMO** [comment]
Could you please indicate which version of Artisan you are using on which platform and share this .alog file by renaming it to .txt as well as you artisan-settings.aset file (save via menu Help >> Save Settings) renamed to artisan-settings.txt file and drag-drop both here to an answer field on GitHub? 

Thanks a lot! We will investigate.
